projects/tinkerbell/charts/Makefile

```diff
@@ -0,0 +1,31 @@
+BASE_DIRECTORY:=$(abspath ../../../)
```

Comment:
why is this a separate project? in other projects with helm charts we include it in the main project build